atm/VOLKOV/src/main/java/com/mybank/atm/controller/DataBase.java

```diff
@@ -0,0 +1,29 @@
+package com.mybank.atm.controller;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class DataBase {
```

Comment:
Consider using a singleton patter instead of static methods to create and get a Connection object.